xcm/src/v2/mod.rs

```diff
@@ -885,48 +897,81 @@ impl<RuntimeCall> Instruction<RuntimeCall> {
 
 // TODO: Automate Generation
 impl<RuntimeCall, W: XcmWeightInfo<RuntimeCall>> GetWeight<W> for Instruction<RuntimeCall> {
-	fn weight(&self) -> Weight {
+	fn weight(&self) -> sp_weights::Weight {
```

Comment:
Why is this being changed? Other than conversion from XCM v3, XCM v2 module should remain the same, no?

Reply:
The `GetWeight` trait has changed, so this implementation would unfortunately have to change accordingly as well.